source/class/qx/ui/core/MPlacement.js

```diff
@@ -376,26 +377,32 @@ qx.Mixin.define("qx.ui.core.MPlacement", {
         );
       }
 
-      var coords =
+      const coords =
         target.getContentLocation() || this.getLayoutLocation(target);
+      const size = this.__getPlacementSize();
```

Comment:
This is a particularly good example of why this use of `const` is inappropriate - the object in `size` is not constant at all.

Reply:
As suggested, I've changed it to `var`.